CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Native function calling support to `OpenAiChatPromptDriver`, `AzureOpenAiChatPromptDriver`, `AnthropicPromptDriver`, `AmazonBedrockPromptDriver`, `GooglePromptDriver`, and `CoherePromptDriver`.
 - `OllamaEmbeddingDriver` for generating embeddings with Ollama.
 - `GriptapeCloudKnowledgeBaseVectorStoreDriver` to query Griptape Cloud Knowledge Bases.
+- `GenericArtifact` for storing any data.
```

Comment:
I don't feel strongly about it, but what about `AnyArtifact` since it is similar to the `Any` python type?

Reply:
Now I vote for `AnyArtifact` since we both had the same thought independently.

Reply:
An argument against `AnyArtifact` is that it's very python-ey whereas the other artifact types are intentionally not. I.e. 
- `TextArtifact` -> `value: str`
- `BlobArtifact` -> `value: bytes`
- etc

Reply:
Let's see if @vasinov or @emjay07 have any thoughts. @emjay07 is going to play with it tomorrow so that may lead to some naming insight.